### DISCUSSION TITLE
How to make tooltip work on Tab focus?

### DISCUSSION THREAD

**Discussion by dorin-js** [mantinedev/mantine#2203] [Q&A] [answered]
Currently tooltip works only on hover. But say, i have a button or action icon, its not clear what is it for. Keyboard users will never know. Is there a way to implement it or would it be good to place it in Future requests?

**Comment by rtivital** [comment, marked as answer]
https://mantine.dev/core/tooltip/#change-events

**Reply by dorin-js** [reply]
Many thanks! blind me :smile:
